chain/api/src/executors.rs

```diff
@@ -33,8 +33,8 @@ pub struct RpcEthereumClientConfig {
     ///
     /// This must be strictly greater than any timeouts in the underlying `HoprRpcOperations`
     ///
```

Comment:
What about this part? Is it? Should there be a check?

Reply:
It is actually not a must.

Reply:
Then let's fix the comment. 🍻